### PR TITLE
Niloofar / Refactor deriv-go page [Translation Key Extraction]

### PR DESCRIPTION
**Pull request by niloofar-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/elements/roadmap.tsx [hunk 1]
```diff
@@ -63,23 +73,16 @@ const StyledButton = styled(LinkButton)`
...
-    }
-}
 const Roadmap = ({ portal }: RoadmapProps) => {
     return (
         <StyledSectionContainer>
             <Container>
                 <Flex width="1202px" fd="column" ai="center" jc="center">
                     <StyledHeader as="h2" type="heading-2" align="center">
-                        <Localize translate_text="What’s next?" />
+                        <Localize translate_text="_t_What’s next?_t_" />
```
Comment (mohammad-hashemi-deriv) [inline src/components/elements/roadmap.tsx:82]: Did we decide to store short strings directly inside `translation_text` prop?

**Review by shayan-deriv — APPROVED** [review]
there are some lint errors. please check them :+1:

**Comment by niloofar-deriv** [comment]
> there are some lint errors. please check them +1

These errors are out of the scope of this card and changes, will fix them later, 
Also we don't have these errors when I create a PR to the master branch it's just in my fork

File: src/components/elements/roadmap.tsx (hunk 1 above)
Comment (niloofar-deriv) [reply src/components/elements/roadmap.tsx:82]: @mohammad-hashemi-deriv we changed the key extractions, right now we can directly passing the texts to the `localize` component and function without defining a variable for them!

**Review by mohammad-hashemi-deriv — APPROVED** [review]

File: src/pages/deriv-go/_deriv-go-content.tsx [hunk 2]
```diff
@@ -70,16 +82,17 @@ const Row = styled(Flex)<{ margin_right: string }>`
...
+
     return (
         <SectionContainer
             p="8.9rem 0 8rem 0"
             background="rgba(243, 246, 250, 1)"
             tabletL={{ pt: '40px' }}
         >
             <StyledContainer>
-                {P2P.map((item, index) => {
+                {P2P.map(({ image_alt, image_name, subtitle, title }, index) => {
```
Comment (mohammad-hashemi-deriv) [inline src/pages/deriv-go/_deriv-go-content.tsx:95]: Let's remove `index` as `key`

File: src/pages/deriv-go/_other-apps.tsx [hunk 3]
```diff
@@ -112,28 +111,31 @@ const OtherApps = () => {
...
+                                is_eu
+                                    ? '_t_Check out our other app_t_'
+                                    : '_t_Check out our other apps_t_'
+                            }
+                        />
+                    </StyledHeader>
+
                     <Flex tablet_direction="column" tablet_ai="center" mt="40px">
-                        {other_apps.map((item, index) => {
+                        {other_apps.map(({ content, id, image_alt, title, url, icon }, index) => {
```
Comment (mohammad-hashemi-deriv) [inline src/pages/deriv-go/_other-apps.tsx:125]: Same here.

**Review by mohammad-hashemi-deriv — APPROVED** [review]

**Review by mohammad-hashemi-deriv — APPROVED** [review]

**Pull request merged**